integrationtest/dmqfull/main.d

```diff
@@ -27,7 +27,7 @@ import ocean.io.Stdout;
 
 static immutable RECORD_COUNT = 10000;
 
-version (UnitTest) {} else
+version ( unittest ) {} else
```

Comment:
Shouldn't it be `version (unittest)` ?

Reply:
You mean in terms of formatting? I changed every instance in the entire repo to`"version ( unittest )` since that seemed to be the dominant formatting. There were only a couple of cases like this one where it didn't have whitespace.

Reply:
When I did the change in Ocean, I went with `version (unittest)` everywhere for the same reasons, hence why I raised the point.

Reply:
I am going to merge as I am not too concerned about this slight change in formatting.